frontend/src/components/CodeIDE/CodeIDEButtons.tsx

```diff
@@ -66,6 +75,7 @@ export const CodeIDEButtons = (props: CodeIDEButtonProps) => {
     shallow,
   )
   const { isOpen, onToggle } = useDisclosure()
```

Comment:
This should be renamed to describe what modal it affects (like in the line below), as there are two of them.

Reply:
Updated